app/db/models/index.js

```diff
@@ -506,24 +497,20 @@ module.exports = function (sequelize, env) {
     },
 
     compare: function (pass, user, callback) {
-      bcrypt.compare(pass, user.password.saltedHash, function (err, res) {
-        callback(err, res);
-      });
+      bcrypt.compare(pass, user.Password.saltedHash, callback);
     },
 
     createOauthLogin: function (userId, clientId, callback) {
       oauthClient.findOrCreate({
-        client: clientId
-      }, {
-        client: clientId
-      }).then(function (client) {
-        return oauthLogin.create({
+          client: clientId
+        }, {
+          client: clientId
+        }).then(client => oauthLogin.create({
```

Comment:
does this work without `spread`? Won't `findOrCreate` return an array instead so this won't work?